src/servicebus/azext_servicebus/_validators.py

```diff
@@ -0,0 +1,129 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
```

Comment:
A lot of these validators are not need as you can use `type=int` for example on the parameters themselves in your _params.py.
https://docs.python.org/3/library/argparse.html#type

Reply:
removed the validators for bool and int